pom.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+<!--suppress DeprecatedClassUsageInspection, DeprecatedClassUsageInspection, DeprecatedClassUsageInspection, DeprecatedClassUsageInspection, DeprecatedClassUsageInspection -->
```

Comment:
do we really need this suppression to be placed 5 times ?
It should be covered by "Production" scope usage.